src/data/mainnet/base-tokens-info.json

```diff
@@ -20,5 +20,158 @@
     "minimumAppVersionToSwap": "1.81.0",
     "isCashInEligible": true,
     "isStableCoin": true
+  },
+  {
+    "name": "Cartesi Token",
+    "symbol": "CTSI",
+    "decimals": 18,
+    "address": "0x259fac10c5cbfefe3e710e1d9467f70a76138d45",
+    "imageUrl": "https://raw.githubusercontent.com/valora-inc/address-metadata/main/assets/tokens/CTSI.png",
+    "isNative": false,
+    "infoUrl": "https://www.coingecko.com/en/coins/cartesi"
+  },
+  {
+    "name": "Curve.Fi USD Stablecoin",
+    "symbol": "crvUSD",
+    "decimals": 18,
+    "address": "0x417ac0e078398c154edfadd9ef675d30be60af93",
+    "imageUrl": "https://raw.githubusercontent.com/valora-inc/address-metadata/main/assets/tokens/crvUSD.png",
```

Comment:
@jh2oman is there an associated asset for this token?

Reply:
The file already exists! same one from another chain